pkg/vpcmodel/unifyGrouping.go

```diff
@@ -0,0 +1,33 @@
+package vpcmodel
+
+type cacheGroupedElements struct {
+	groupedEndpointsElemsMap map[string]*groupedEndpointsElems
+	groupedExternalNodesMap  map[string]*groupedExternalNodes
+}
+
+func initCacheGroupedElem() *cacheGroupedElements {
+	return &cacheGroupedElements{
+		map[string]*groupedEndpointsElems{},
+		map[string]*groupedExternalNodes{},
+	}
+}
+
+// unifyMultiVPC unifies multi-vpc graph for endpoints and subnets connectivity s.t.
+// each node appears once across multi-vpcs this is relevant only for DRAWIO
+// in which there is a multivpc presentation
+func unifyMultiVPC(config1 map[string]*VPCConfig, nodesConn map[string]*VPCConnectivity,
+	subnetsConn map[string]*VPCsubnetConnectivity, uc OutputUseCase) {
+	cacheGrouped := initCacheGroupedElem()
+	for vpcName := range config1 {
```

Comment:
The keys in the map `config1 map[string]*VPCConfig` are VPC UIDs and not names (as documented in `parser.go` function `VPCConfigsFromResources()` ), so better rename var `vpcName` to `vpcUID`

Reply:
done